addOns/graphql/src/main/java/org/zaproxy/zap/extension/graphql/ExtensionGraphQL.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.graphql;
+
+import java.io.File;
+import java.util.List;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.CommandLineArgument;
+import org.parosproxy.paros.extension.CommandLineListener;
+import org.parosproxy.paros.extension.ExtensionAdaptor;
+import org.parosproxy.paros.extension.ExtensionHook;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.view.ZapMenuItem;
+
+public class ExtensionGraphQL extends ExtensionAdaptor implements CommandLineListener {
```

Comment:
I'd suggest `GraphQl`, having acronyms all uppercase makes it harder to read when there are several together (e.g. `GraphQLAPI`).